src/android/src/main/java/com/reactnativecommunity/slider/ReactSlider.java

```diff
@@ -233,4 +234,23 @@ public void setThumbImage(final String uri) {
       setThumb(getThumb());
     }
   }
+    @Override
+    public boolean onTouchEvent(MotionEvent event) {
+        //设置父容器不拦截事件
+        if (getParent() != null) {
+            try {
+                getParent().requestDisallowInterceptTouchEvent(true);
+                //SeekBar 放在其他视图中，手指放在 SeekBar 上面不会立即触发滑动操作
```

Comment:
The code should always be self-explanatory.
If comments are required for the code to be more understandable they should be written in English.

Applies also to line 239.